examples/cra-kitchen-sink/src/stories/index.js

```diff
@@ -110,6 +111,8 @@ storiesOf('Button', module)
     const salutation = nice ? 'Nice to meet you!' : 'Leave me alone!';
     const dateOptions = { year: 'numeric', month: 'long', day: 'numeric' };
 
+    button('Arbitrary action', action('You clicked it!'));
```

Comment:
This example is rather meaningless right? calling `action` from a button inside the knobs panel, won't be useful, or am I missing something here?

Could you add an example that would help understand the true use-case of this change? 👍

Reply:
Totally reasonable

Reply:
So I made an example in the CRA for a case where i've normally included a button _inside_ the story itself to trigger something like loading some external data through a render callback component, or generating a set of random data. These are the kind of use cases I'd have in mind for something like this. 

I haven't thought too hard about it, there could be better ways to do it. 

If you think that's not a good example, or not within knobs paradigm—totally fine. Don't have my heart set on it :)